packages/astro/src/assets/fonts/providers/google.ts

```diff
@@ -0,0 +1,10 @@
+import { defineFontProvider } from '../helpers.js';
+
+export const GOOGLE_PROVIDER_NAME = 'google';
+
+export function google() {
+	return defineFontProvider({
+		name: GOOGLE_PROVIDER_NAME,
+		entrypoint: 'astro/assets/fonts/providers/google',
```

Comment:
I think the paths of built-in providers are wrong. They should be `dist/assets/fonts/providers/..`

Reply:
Presumably they will be resolved as imports, and `astro/assets/fonts/*` could be added to exports

Reply:
The entrypoint is used as a placeholder for now, to try figure out the API. I'll address it in the following PR